.ci/setup_env_opensuse.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || source "/usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+echo "Install chronic"
+sudo -E zypper -n install moreutils
+
+echo "Install curl"
+sudo -E zypper -n install curl
+
+echo "Install git"
+sudo -E zypper -n install git
+
+echo "Install kata containers dependencies"
+chronic sudo -E zypper -n install libtool automake autoconf bc perl-Alien-SDL libpixman-1-0-devel coreutils
+
+echo "Install qemu dependencies"
+chronic sudo -E zypper -n install libcap-devel libattr1 libcap-ng-devel librbd-devel
+
+echo "Install kernel dependencies"
+chronic sudo -E zypper -n install libelf-devel flex
+
+echo "Install CRI-O dependencies"
+chronic sudo -E zypper -n install libglib-2_0-0 libseccomp-devel libapparmor-devel libgpg-error-devel python2-pkgconfig \
+	go-md2man glibc-devel-static thin-provisioning-tools libgpgme-devel libassuan-devel glib2-devel glibc-devel libcontainers-common
+
+echo "Install bison binary"
+chronic sudo -E zypper -n install bison
+
+echo "Install libudev-dev"
+chronic sudo -E zypper -n install libudev-devel
+
+echo "Install Build Tools"
+chronic sudo -E zypper -n install patterns-devel-base-devel_basis python pkg-config zlib-devel
+
+echo -e "Install CRI-O dependencies available"
+chronic sudo -E zypper -n install libdevmapper1_03 util-linux
```

Comment:
can this be moved to L33? with the other cri-o dependencies.
Same below when installing `libostree-devel`